<!--
  Standard launch file. for more information check http://wiki.ros.org/roslaunch/XML.
 -->
<launch>

  <!-- Args -->
  <arg name="config_override"        default="$(find ros1_pytemplate)/config/fibonacci-override.yaml" doc="used by upper launchers to customize configuration"/>

  <!-- Launch oracle node -->
  <node pkg="ros1_pytemplate" type="fibonacci_pub_node.py" name="fibonacci_pub">
    <rosparam command="load" file="$(find ros1_pytemplate)/config/fibonacci-default.yaml" />
    <rosparam command="load" file="$(arg config_override)" />
  </node>

  <!-- Launch follower node -->
  <node pkg="ros1_pytemplate" type="echo.sh" name="fibonacci_sub">
    <remap from="/fibonacci_sub/fibonacci" to="/fibonacci_pub/fibonacci" />
  </node>

</launch>
